march_rqt_note_taker/test/entry_model_test.py

```diff
@@ -14,10 +15,20 @@ def test_empty_model(self):
         self.assertEqual(self.model.rowCount(), 0)
         self.assertEqual(self.model.columnCount(), 2)
 
+    def test_horizontal_model_header(self):
+        self.assertEqual(self.model.headerData(0, Qt.Horizontal, Qt.DisplayRole), 'Time')
+        self.assertEqual(self.model.headerData(1, Qt.Horizontal, Qt.DisplayRole), 'Entry')
+
     def test_insert_single_row(self):
-        self.model.insert_row(Entry('test'))
+        entry = Entry('test', QDateTime.fromSecsSinceEpoch(5))
+        self.model.insert_row(entry)
         self.assertEqual(self.model.rowCount(), 1)
 
+        time_index = self.model.createIndex(0, 0)
+        content_index = self.model.createIndex(0, 1)
+        self.assertEqual(self.model.data(time_index), entry.time_string())
+        self.assertEqual(self.model.data(content_index), entry.content)
```

Comment:
Should you not limit tests to one assert per test?

Reply:
I will move the bottom part that I added to a separate test.